test/test_rake_task.rb

```diff
@@ -296,24 +296,26 @@ def test_all_prerequisite_tasks_includes_self_on_cyclic_dependencies
     assert_equal [a, b], a.all_prerequisite_tasks.sort_by { |t| t.name }
   end
 
-  def test_timestamp_returns_now_if_all_prereqs_have_no_times
+  def test_timestamp_returns_now_regardless_prereq_times
```

Comment:
Previous test name was misleading because current behavior is to have time stamp always report as now.